source_servers/mordhau/README.md

```diff
@@ -4,11 +4,11 @@ MORDHAU is a medieval first & third person multiplayer slasher. Enter a hectic b
 
 
 ### Server Ports
-Mordhau requires 2 ports
+Mordhau requires 3 ports
 
 game port (default 7777)
```

Comment:
all I can find says it needs 7777 and 7778

Reply:
Also needs 4380 for some reason.

Reply:
I'm not using 4380, so not sure what it is for.  

The beacon port was added in this patch https://steamcommunity.com/games/629760/announcements/detail/1617266961416483617